truffle/docs/Languages.md

```diff
@@ -10,6 +10,7 @@ This page is intended to keep track of the growing number of Truffle language im
 * [TruffleSOM](https://github.com/SOM-st/TruffleSOM), a SOM Smalltalk implementation. 
 * [SOMns](https://github.com/smarr/SOMns), a Newspeak implementation for Concurrency Research.
 * [SimpleLanguage](https://github.com/graalvm/simplelanguage), a toy language implementation to demonstrate Truffle features. 
+* [TRegex](https://github.com/oracle/graal/tree/master/regex), an implementation of a subset of ECMAScript regular expressions.
```

Comment:
@wirthi what do you think about

```suggestion
* [TRegex](https://github.com/oracle/graal/tree/master/regex), an internal available for use by other languages implementation of a subset of ECMAScript regular expressions.
```

?

Reply:
maybe: "An implementation of a subset of ECMAScript regular expressions (internal i.e. only for use by other languages.)